### DISCUSSION TITLE
Connecting Electric to local Supabase dev setup

### DISCUSSION THREAD

**Discussion by marlou-lopez** [electric-sql/electric#765] [Questions] [answered]
Hello, I'm trying to setup Electric locally and connecting it to Supabase that's also running on local.

After initializing Supabase 
![image](https://github.com/electric-sql/electric/assets/35794059/e04bd9ef-c3e4-4733-a111-8f6cac5f1dca)

I tried to run the sync service following this guide: [configuring-electric-to-connect-to-supabase](https://electric-sql.com/docs/integrations/deployment/supabase#4-configuring-electric-to-connect-to-supabase) but it cannot connect to the postgres service.
![image](https://github.com/electric-sql/electric/assets/35794059/96e5444b-c457-490c-8b48-2263190c9f75)

Following the supabase guide on [configuring-electric-to-connect-to-supabase](https://electric-sql.com/docs/integrations/deployment/supabase#4-configuring-electric-to-connect-to-supabase) works using an actual supabase project. Are there steps I'm missing in order to connect to a local supabase setup?

**Comment by davidmartos96** [comment, marked as answer]
I believe the issue could be that you are using 127.0.0.1 for the database url in the electric container. From the container point of view, that port is not accessible on that host.
You can try changing the IP to the IP of your machine (192.168.x.x)

**Reply by marlou-lopez** [reply]
It worked. Thank you!

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
